web/src/user_card_popover.ts

```diff
@@ -567,17 +588,8 @@ function toggle_sidebar_user_card_popover($target: JQuery): void {
     const user_id = elem_to_user_id($target);
     const user = people.get_by_user_id(user_id);
 
-    // Hiding popovers may mutate current_user_sidebar_user_id.
-    const previous_user_sidebar_id = current_user_sidebar_user_id;
-
     // Hide popovers
     hide_all();
-
-    if (previous_user_sidebar_id === user_id) {
```

Comment:
Why is this code removed? This breaks the behavior of closing the opened popover if I click on the three dot icon.